vision/cloud-client/detect/beta_snippets.py

```diff
@@ -272,7 +273,75 @@ def detect_document_features_uri(gcs_uri):
                         for symbol in word.symbols:
                             print('\t\t\tSymbol: {} (confidence: {})'.format(
                                 symbol.text, symbol.confidence))
-# [END vision_fulltext_detection_pdf_gcs_beta]
+# [END detect_batch_annotate_files_gcs_beta]
+
+
+# [START detect_async_batch_annotate_images_beta]
+def async_batch_annotate_images_uri(image_uri, gcs_uri):
```

Comment:
`input_image_uri` and `output_uri` might help people distinguish, since both are gcs links, according to the description below.

Reply:
How strongly do you feel about this? The other methods I see just have 'uri' for the output.

Reply:
The other ones use `uri` as input

Reply:
Done